src/lib.rs

```diff
@@ -138,6 +138,18 @@ pub struct ChildStream<Sout, Serr> {
 }
 }
 
+impl<Sout, Serr> ChildStream<Sout, Serr> {
+    /// Return a mutable reference to the child object
```

Comment:
The reference returned by this function is not mutable

Reply:
Thanks @lpenz, good catch. That's what I get for copy/paste...

Fixed.